src/libfuncs/bytes31.rs

```diff
@@ -69,7 +69,7 @@ pub fn build_const<'ctx, 'this>(
         Attribute::parse(context, &format!("{value} : {value_ty}")).unwrap(),
         location,
     ));
-    entry.append_operation(helper.br(0, &[op0.result(0)?.into()], location));
+    entry.append_op_result(helper.br(0, &[op0.result(0)?.into()], location));
```

Comment:
you shouldn't use `append_op_result` for operations which don't have results, it was correct before. same with all the others